store/hbase/kv.go

```diff
@@ -136,21 +139,26 @@ func (s *hbaseStore) CurrentVersion() (kv.Version, error) {
 type Driver struct {
 }
 
-// Open opens or creates an HBase storage with given dsn, format should be 'zk1,zk2,zk3|tsoaddr:port/tblName'.
-// If tsoAddr is not provided, it will use a local oracle instead.
-func (d Driver) Open(dsn string) (kv.Storage, error) {
+// Open opens or creates an HBase storage with given path.
+//
+// The format of path should be 'hbase://zk1,zk2,zk3/table[?tso=host:port]'.
+// If tso is not provided, it will use a local oracle instead. (for test only)
+func (d Driver) Open(path string) (kv.Storage, error) {
 	mc.mu.Lock()
 	defer mc.mu.Unlock()
 
-	if store, ok := mc.cache[dsn]; ok {
-		// TODO: check the cache store has the same engine with this Driver.
-		return store, nil
-	}
-
-	zks, oracleAddr, tableName, err := parseDSN(dsn)
+	zks, oracleAddr, tableName, err := parsePath(path)
 	if err != nil {
 		return nil, errors.Trace(err)
 	}
+	uuid := fmt.Sprintf("hbase-%v-%v", zks, tableName)
+
```

Comment:
Yeah, it is not strict to use zks. If we want to identify hbase accurately, we need to load hbase clusterID from zookeeper. We may create an issue and fix it later.